x-pack/plugins/endpoint/scripts/resolver_generator.ts

```diff
@@ -35,13 +35,15 @@ async function main() {
     eventIndex: {
       alias: 'ei',
       describe: 'index to store events in',
+      // This should be changed to events-endpoint-default-1 when this issue gets resolved:
+      // https://github.com/elastic/endpoint-app-team/issues/311
```

Comment:
would you remove this commend and instead link to this line of code from the issue?